### PR TITLE
hardening/893_add_admin_GET_all_parameters

### PR DESCRIPTION
**Pull request by pcoello25**
* Partially implements issue #893 
  * Update CHANGES_NEXT_RELEASE is neccesary (first PR related with this issue)
* 100% Unit tests passed: 
```
Results :  Tests run: 67,   Failures: 0, Errors: 0, Skipped: 0 [cygnus-common]
Results :  Tests run: 106, Failures: 0, Errors: 0, Skipped: 0 [cygnus-ngsi]
```
* e2e (unofficial) tests passed:
```
$ curl -X GET http://localhost:8081/admin/configuration/agent/cygnus/apache-flume-1.4.0-bin/conf/mysql_confs/my_agent.conf
{"success":"true","result" : {
{"agent":{"cygnusagent.sinks.mysql-sink.mysql_port":"3306","channels.mysql-channel.transactionCapacity":"100","cygnusagent.sources.http-source.interceptors.gi.type":"com.interceptors.NGSIGroupingInterceptor$Builder","cygnusagent.sources.http-source.handler.default_service":"def_service","cygnusagent.sources.http-source.interceptors":"ts gi","cygnusagent.sinks.mysql-sink.mysql_host":"localhost","cygnusagent.sources.http-source.type":"org.apache.flume.source.http.HTTPSource","cygnusagent.sources.http-source.handler.default_service_path":"\/def_servpath","cygnusagent.sources.http-source.handler.notification_target":"\/notify","cygnusagent.sinks.mysql-sink.enable_grouping":"false","cygnusagent.sinks.mysql-sink.mysql_password":"root","cygnusagent.sources":"http-source","cygnusagent.channels.mysql-channel.capacity":"1000","cygnusagent.sinks.mysql-sink.type":"com.NGSIMySQLSink","cygnusagent.sources.http-source.interceptors.ts.type":"timestamp","cygnusagent.sinks.mysql-sink.batch_timeout":"10","cygnusagent.sources.http-source.port":"5050","cygnusagent.sinks.mysql-sink.batch_size":"1","cygnusagent.sinks.mysql-sink.mysql_username":"root","cygnusagent.sinks.mysql-sink.channel":"mysql-channel","cygnusagent.sources.http-source.interceptors.gi.grouping_rules_conf_file":"\/cygnus\/apache-flume-1.4.0-bin\/conf\/grouping_rules_2.conf","cygnusagent.channels.mysql-channel.type":"memory","cygnusagent.sources.http-source.channels":"mysql-channel","cygnusagent.sources.http-source.handler.events_ttl":"2","cygnusagent.sources.http-source.handler":"com.handlers.NGSIRestHandler","cygnusagent.sinks.mysql-sink.data_model":"dm-by-service-path","cygnusagent.sinks":"mysql-sink","cygnusagent.sinks.mysql-sink.attr_persistence":"row","cygnusagent.channels":"mysql-channel"}}}

$ curl -X GET http://localhost:8081/v1/admin/configuration/agent/home/pcoello25/cygnus/apache-flume-1.4.0-bin/conf/mysql_confs/agent_020.conf
{"success":"true","result" : {
{"agent":{"cygnusagent.sinks.mysql-sink.mysql_port":"3306","cygnusagent.channels.mysql-channel.transactionCapacity":"100","cygnusagent.sources.http-source.interceptors.gi.type":"com.telefonica.iot.cygnus.interceptors.NGSIGroupingInterceptor$Builder","cygnusagent.sources.http-source.handler.default_service":"def_service","cygnusagent.sources.http-source.interceptors":"ts gi","cygnusagent.sinks.mysql-sink.mysql_host":"localhost","cygnusagent.sources.http-source.type":"org.apache.flume.source.http.HTTPSource","cygnusagent.sources.http-source.handler.default_service_path":"\/def_servpath","cygnusagent.sources.http-source.handler.notification_target":"\/notify","cygnusagent.sinks.mysql-sink.enable_grouping":"false","cygnusagent.sinks.mysql-sink.mysql_password":"root","cygnusagent.sources":"http-source","cygnusagent.channels.mysql-channel.capacity":"1000","cygnusagent.sinks.mysql-sink.type":"com.telefonica.iot.cygnus.sinks.NGSIMySQLSink","cygnusagent.sources.http-source.interceptors.ts.type":"timestamp","cygnusagent.sinks.mysql-sink.batch_timeout":"10","cygnusagent.sources.http-source.port":"5050","cygnusagent.sinks.mysql-sink.batch_size":"1","cygnusagent.sinks.mysql-sink.mysql_username":"root","cygnusagent.sinks.mysql-sink.channel":"mysql-channel","cygnusagent.sources.http-source.interceptors.gi.grouping_rules_conf_file":"\/home\/pcoello25\/cygnus\/apache-flume-1.4.0-bin\/conf\/grouping_rules_2.conf","cygnusagent.channels.mysql-channel.type":"memory","cygnusagent.sources.http-source.channels":"mysql-channel","cygnusagent.sources.http-source.handler.events_ttl":"2","cygnusagent.sources.http-source.handler":"com.telefonica.iot.cygnus.handlers.NGSIRestHandler","cygnusagent.sinks.mysql-sink.data_model":"dm-by-service-path","cygnusagent.sinks":"mysql-sink","cygnusagent.sinks.mysql-sink.attr_persistence":"row","cygnusagent.channels":"mysql-channel"}}}
```
* Assignee: @frbattid

### CODE REVIEW

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 1]
```diff
@@ -597,6 +603,42 @@ protected void handleGetSubscriptions(HttpServletRequest request, HttpServletRes
         }
         
     } // handleGetSubscriptions
+    
+    protected void handleGetAdminParameters(HttpServletRequest request, HttpServletResponse response, boolean v1)
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: handleGetAdminParameters --> handleGetAdminConfigurationAgent

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 2]
```diff
@@ -597,6 +603,42 @@ protected void handleGetSubscriptions(HttpServletRequest request, HttpServletRes
...
         
     } // handleGetSubscriptions
+    
+    protected void handleGetAdminParameters(HttpServletRequest request, HttpServletResponse response, boolean v1) 
+            throws IOException {
+        
+        response.setContentType("json;charset=utf-8");
+        String pathToFile;
+        
+        if (v1) {
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Cool :) NTC

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java (hunk 1 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Fixed in b4285b4

**Comment by frbattid** [comment]
LGTM

**Pull request merged**